test/models/podcast_import_test.rb

```diff
@@ -71,6 +78,7 @@
   end
 
   it 'creates stories' do
+    importer.set_config_url('http://test.prx.org/transistor_import_config.json')
```

Comment:
Is this necessary for this test?  Wouldn't you get the exact same output if you didn't set a config?